xwiki-commons-core/xwiki-commons-component/xwiki-commons-component-default/src/main/java/org/xwiki/component/annotation/ComponentDescriptorFactory.java

```diff
@@ -26,15 +26,16 @@
 import java.util.ServiceLoader;
 
 import javax.annotation.Priority;
```

Comment:
There is no replacement for `Priority`? Or you just decided not to migrate that yet?

Reply:
I did not really migrate `javax.annotation` yet. It was not really necessary for jakarta application servers.